### PR TITLE
libsm, bump revision after changes to util-macros and xorgproto

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/libsm/libsm-1.2.2.recipe [hunk 1]
```diff
@@ -38,18 +38,23 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	docbook_xml_dtd
+	docbook_xsl_stylesheets
```
Comment (alaviss) [inline x11-libs/libsm/libsm-1.2.2.recipe]: `d` is before `h` so...

```diff
-	haiku${secondaryArchSuffix}_devel
	docbook_xml_dtd
	docbook_xsl_stylesheets
+	haiku${secondaryArchSuffix}_devel
```

Comment (fbrosson) [reply x11-libs/libsm/libsm-1.2.2.recipe]: Yep, but I think there is an implicit rule which allows to put `haiku${secondaryArchSuffix}_devel` first.

Comment (Begasus) [reply x11-libs/libsm/libsm-1.2.2.recipe]: I would prefer haiku on top too ;)

Comment (alaviss) [reply x11-libs/libsm/libsm-1.2.2.recipe]: Well, it's fine if y'all prefer that :)

Comment (Begasus) [reply x11-libs/libsm/libsm-1.2.2.recipe]: Again one of those thing we should have a mutual rule for ;)

**Pull request merged**